.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
Do we really need this file? We can probably just ignore `.husky/_` from the root gitignore.

Reply:
IDK 🤷 I just ran their script